pipelines/templates/install-dependencies.yml

```diff
@@ -53,6 +53,19 @@ steps:
     echo '===== recommended ====='
     python -m pip install -r dependencies/recommended.txt
   displayName: Install Python dependencies
+  condition: and(succeeded(), not(contains('${{ parameters.platform }}', 'gpu')))
+
+- script: |
```

Comment:
For legacy I used `mv dependencies/recommended_legacy.txt dependencies/recommended.txt`.
I can't remember the reason though... Maybe some sort of debugability.